modules/custom/social_magic_login/src/Controller/MagicLoginController.php

```diff
@@ -70,7 +75,7 @@ public static function create(ContainerInterface $container) {
    *
    * @see \Drupal\user\Controller\UserController::resetPassLogin
    */
-  public function login($uid, $timestamp, $hash, $destination) {
+  public function login($uid, $timestamp, $hash, $destination): ?RedirectResponse {
```

Comment:
```suggestion
  public function login($uid, $timestamp, $hash, $destination) {
```

We can't use PHP 7.0 code yet.